hw/top_earlgrey/ip/clkmgr/data/autogen/clkmgr.hjson

```diff
@@ -457,8 +457,8 @@
         },
       ]
     },
-
-      { name: "IO_MEASURE_CTRL",
+  
```

Comment:
Seems like two extra spaces, remove?

Reply:
sorry do you mean the extra new line? 
i think that might be an artifact of mako... when i take out the `\` at the end, i get an extra line.
If i have that, then I get the two space..

Reply:
o wait you meant above in the empty space. yes will do.